apiary.apib

```diff
@@ -14014,4 +14014,621 @@ Competitor keywords(s) from BirdEye platform can be fetched using following filt
+}
+
+<div class="paramsTableWrap">
+    <table class="paramsTable">
+        <caption>Response Body</caption>
+        <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            <td>id</td>
+            <td>id</td>
+        </tr>
+        <tr>
+            <td>sourceId</td>
+            <td>Source Id</td>
+        </tr>
+         <tr>
+            <td>businessId</td>
+            <td>Business ID</td>
+        </tr>
+        <tr>
+            <td>sourceUrl</td>
+            <td>Source Url</td>
+        </tr>
+        <tr>
+            <td>sourceName</td>
+            <td>Aggregation Source Name</td>
+        </tr>
+        <tr>
+            <td>thumbnailUrl</td>
+            <td>Thumbnail Url</td>
+        </tr>
+        <tr>
+            <td>backGroundColor</td>
+            <td>BackGround Color</td>
+        </tr>
+        <tr>
+            <td>textColor</td>
+            <td>TextColor</td>
+        </tr>
+        <tr>
+            <td>userName</td>
+            <td>User Name</td>
+        </tr>
+         <tr>
+            <td>password</td>
+            <td>Password</td>
+        </tr>
+         <tr>
+            <td>profileId</td>
+            <td>Profile ID</td>
+        </tr>
+         <tr>
+            <td>reviewFormUrl</td>
+            <td>ReviewFormUrl</td>
+        </tr>
+         <tr>
+            <td>aggregationStatus</td>
+            <td>Aggregation Status</td>
+        </tr>
+         <tr>
+            <td>sourceAlias</td>
+            <td>SourceAlias</td>
+        </tr>
+        <tr>
+            <td>disabledAggregation</td>
+            <td>disabledAggregation</td>
+        </tr>
+        <tr>
+            <td>disabledAggregationReason</td>
+            <td>disabledAggregationReason</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+
+<div class="paramsTableWrap">
+    <table class="paramsTable">
+        <caption>Error Codes</caption>
+        <thead>
+        <tr>
+            <th>HTTP Status</th>
+            <th>BirdEye error code</th>
+            <th>Description</th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr>
+            <td>401 Unauthorized</td>
+            <td>1161</td>
+            <td>Invalid API key</td>
+        </tr>
+        <tr>
+            <td>429 Too Many Requests</td>
+            <td>89</td>
+            <td>Rate limit exceeded</td>
+        </tr>
+        <tr>
+            <td>404 Not Found</td>
+            <td>1011</td>
+            <td>Business id is invalid</td>
+        </tr>
+        <tr>
+            <td>401 Unauthorized</td>
+            <td>1167</td>
+            <td>API key is missing</td>
+        </tr>
+        </tbody>
+    </table>
+</div>
+
++ Request (application/json)
+
+    + Headers
+
+            Accept: application/json
+
++ Response 200
+
+    + Body
+
+{
+    "id": 140179308,
+    "sourceId": 2,
+    "businessId": 100037706,
```

Comment:
@khyati-gupta is this long business number or short business id? we should not return short business id

Reply:
@ameyavirkar : This is short businessId which is getting returned from create competitor API and is being used in creating the enterprise location. Also  as discussed with Sachin Gupta it has been added in GET competitor API as well. 
 If we remove it here then  above APIs will also need code changes as well. Please advise.